ocaml/doc/wire-protocol.md

```diff
@@ -170,7 +170,7 @@ to subequent RPC calls as an authentication token.
 A session can be terminated with the `session.logout` function:
 
 ```python
-   void  session.logout(session_id session)
+   void  session.logout(session ref)
```

Comment:
To be honest, I'd rather we had used a completely different name for this param; `session_id` can be very easily read or understood as `session_uuid`, but in reality it's a ref, which causes confusion (see CA-40854). But it may be a bit too late to change it?